javascript/ql/src/semmle/javascript/security/dataflow/TaintedPathCustomizations.qll

```diff
@@ -12,19 +12,17 @@ module TaintedPath {
    */
   abstract class Source extends DataFlow::Node {
     /** Gets a flow label denoting the type of value for which this is a source. */
-    DataFlow::FlowLabel getAFlowLabel() {
-      result instanceof Label::PosixPath
-    }
+    DataFlow::FlowLabel getAFlowLabel() { result instanceof Label::PosixPath }
   }
 
   /**
    * A data flow sink for tainted-path vulnerabilities.
    */
   abstract class Sink extends DataFlow::Node {
+    Sink() { not this instanceof Sanitizer }
```

Comment:
This seems out of place for this PR, and if we want it, then it is perhaps something that should be implemented in `Configuration.qll` as something like `isReallySink(Node sink) { isSink(sink) and not isBarrier(sink) }`

Reply:
It was part of my testing for the PR and it should have been removed, but I forgot.